source/lumars/state.d

```diff
@@ -783,7 +927,14 @@ struct LuaState
             static foreach (i, p; params)
             {
                 idx = cast(int)(i - params.length);
-                if (this.isType!(params[i])(idx))
+                if (params.length != lua_gettop(this.handle))
+                {
+                    static if(i == 0)
+                    {
+                        return getAsStruct!(T, T.fieldNames)(index); 
+                    }
+                }
```

Comment:
This seems unrelated - is it another leftover?

Reply:
If I push a Tuple, I cannot get it back without this change. See unittest at line 1300.